spaces/domains.json

```
@@ -364,4 +364,5 @@
   "vote.nft-gods.com": "nft-gods.eth",
   "vote.sunrisegaming-dao.com": "sunrisegaming.eth",
   "dao.babyspacefloki.com": "babyspacefloki.eth"
+  "vote.mgh.dao": "metagamehub.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dao.babyspacefloki.com": "babyspacefloki.eth",
  "vote.mgh.dao": "metagamehub.eth"
```